packages/tools/guiEditor/src/diagram/workbench.tsx

```diff
@@ -1025,6 +1027,19 @@ export class WorkbenchComponent extends React.Component<IWorkbenchComponentProps
         } else if (event.detail) {
             delta = -event.detail;
         }
+
+        this.zooming(1 + delta / 1000);
+    }
+
+    zoomDrag(event: React.MouseEvent) {
+        let delta = 0;
+        if (event.movementY < 0) {
+            delta = -event.movementY;
+        }
+        if (event.movementY > 0) {
+            delta = -event.movementY;
+        }

```

Comment:
I might be missing something, but both of those seem to do the same?

Reply:
You're right! Not sure why I have those as two different conditionals.